Documentation/ExtensionArchitecture/HowTo/UpdateExtensions/UpdateWizards/Creation.rst

```diff
@@ -89,6 +92,25 @@ Method :php:`getPrerequisites()`
             ];
         }
 
+After creating the new upgrade wizard, delete all caches in  
+:guilabel:`Admin tools > Maintanance > Flush TYPO3 and PHP Cache` or via console
+command:
+
+.. tabs::
```

Comment:
We have an include you can use.

Reply:
Where do we have that?

Reply:
https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/blob/main/Documentation/_includes/CliCacheFlush.rst.txt

Reply:
Done

Reply:
Thank you!